resources/views/master.blade.php

```diff
@@ -23,6 +23,28 @@
     @else
     <link rel="stylesheet" href="{{ asset('css/app.css') }}">
     @endif
+
+    @if(config('adminlte.use_ico_only'))
+        <link rel="shortcut icon" href="favicon.ico" />
```

Comment:
I believe the path of favicon.ico should start with / here.

Reply:
Or it should use `asset()` like the link/script-tags